### DISCUSSION TITLE
API Usage

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#1440] [Q&A] [answered]
I have been requested to add support for this plugin in my discord.

I have looked at the API documentation here: https://github.com/TechFortress/GriefPrevention/wiki/API

I want a method to check who has claimed a specific location. Something such as: 

getClaim(Location location)

Is this currently possible?

**Comment by Jikoo** [comment, marked as answer]
You want `DataStore#getClaimAt`. The datastore instance can be obtained from the field `GriefPrevention.datastore`. Most addons probably use the provided static GP instance, i.e. `GriefPrevention.instance.datastore.getClaimAt(params...)`.

Your best bet for learning how to use the API is actually just reverse engineering, honestly. Find an event that you know gets cancelled by GP and trace it back.

**Reply by ghost** [reply]
Thanks! This was exactly what I was looking for.

**Reply by RoboMWM** [reply]
Highly recommended to get the instance from the plugin manager btw.
